infra/bridge/src/utils/exchange-rate.js

```diff
@@ -67,13 +95,14 @@ async function getExchangeRate(market) {
 
     price = await fetchExchangeRate(market)
 
-    if (Number.isNaN(Number(price))) {
-      // API is also down, send back the fallback values
-      // FALLBACK_EXCHANGE_RATE_ETH_USD and FALLBACK_EXCHANGE_RATE_DAI_USD
-      return (
-        process.env[`FALLBACK_EXCHANGE_RATE_${market.replace('-', '_')}`] || 310
-      )
-    }
+    // Disabled Fallback values until we decide on a strategy
```

Comment:
@franckc but where does the constant come from? Also does it need to be periodically updated? Eventually it could possibly be super out of date if I just set it to a price from today.

Reply:
ok I'll just set the constants to a value from todays price.